include/openssl/ocsp.h

```diff
@@ -59,13 +59,30 @@ DECLARE_ASN1_FUNCTIONS(OCSP_REQUEST)
 DECLARE_ASN1_FUNCTIONS(OCSP_SIGNATURE)
 DECLARE_ASN1_FUNCTIONS(OCSP_REQINFO)
 
+// Blocking OCSP request handler: now a special case of non-blocking I/O.
+// Combines OCSP_sendreq_new() with as many calls of OCSP_sendreq_nbio() as
+// needed and then OCSP_REQ_CTX_free(), with a response header maximum line
+// length 4k. It waits indefinitely on a response.
```

Comment:
TIL about util/docs.go and this fails to parse (along with all the other public header file changes): 
```suggestion
// OCSP_sendreq_bio is a blocking OCSP request handler which is a special case of non-blocking I/O.
// |OCSP_sendreq_bio| combines |OCSP_sendreq_new| with as many calls of |OCSP_sendreq_nbio| as
// needed and then |OCSP_REQ_CTX_free|, with a response header maximum line
// length 4k. It waits indefinitely on a response.
```
>As a matter of style, comments should start with the name of the thing that they are commenting on

Also anything within |bars| is converted to a link if it is defined somewhere else, or printed as monospace.

https://github.com/aws/aws-lc/blob/main/util/doc.go#L442-L445